src/sentry/testutils/cases.py

```diff
@@ -749,6 +750,40 @@ def analytics_called_with_args(self, fn, name, **kwargs):
                 return True
         return False
 
+    @contextmanager
+    def api_gateway_proxy_stubbed(self):
+        """Mocks a fake api gateway proxy that redirects via Client objects"""
```

Comment:
```suggestion
        """Monkey patches the api gateway proxy outbound requests, and uses Client objects
        convert those requests into django.test.Client requests.
        """
```

Not sure if this is an improvement, but it took me a minute to figure out what was going on.